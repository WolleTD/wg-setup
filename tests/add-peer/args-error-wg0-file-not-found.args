peer1 iCJSbobpCRHCG2/WP9D1/viGlv+WrNpWtd1XkRzyrFs= 172.16.1.1/32
