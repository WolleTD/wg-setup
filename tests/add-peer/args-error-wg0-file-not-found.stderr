Could not find backing config file for wg0! Do you have permissions to read it\?
